src/components/clusters/NewClusterPage.tsx

```diff
@@ -130,6 +132,10 @@ const NewClusterForm: React.FC<NewClusterFormProps> = ({ pullSecret = '', versio
                       </Text>
                     </TextContent>
                     <InputField ref={nameInputRef} label="Cluster Name" name="name" isRequired />
+                    <SingleNodeCheckbox
+                      name="highAvailabilityModeChkbox"
+                      label="Create Single-Node Cluster"
```

Comment:
Feels like an `a` is missing, i.e. `"Create a Single-Node Cluster"`. 

Also not sure about the capitalization, maybe `"Create a single-node cluster"` ?

WDYT?

Reply:
> Feels like an `a` is missing, i.e. `"Create a Single-Node Cluster"`.
> 
> Also not sure about the capitalization, maybe `"Create a single-node cluster"` ?
> 
> WDYT?

Good point.